department-of-veterans-affairs/GIBCT-Data-Service
# Bah ct111 19025

## Description 
As a developer, I need to update the WEAMS import into GIDS to process the STEM indicator so that the UI can display the STEM information on the school profile page.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19025

## Testing done

1. Run `bundle exec rake db:migrate`
2. Ensure the stem indicator column is present in the database
3. Run `bundle exec rails s -p 4000`
4. Open the GI Bill Comparison Tool Dashboard
5. Export Weam.csv and ensure stem indicator column is generated. 
6. Logged into the GIDS dashboard and exported a copy of the Weam CSV to ensure the stem indicator column was present. 

## Screenshots
![Screen Shot 2019-06-24 at 9 26 03 AM](https://user-images.githubusercontent.com/50601724/60045566-2bc1f080-9693-11e9-9a9b-34c811d58d52.png)


## Acceptance criteria
- [X] The weam.csv with "STEM Indicator" column is imported into GIDS

## Definition of done
